.env.example

```diff
@@ -0,0 +1,15 @@
+RABBITMQ_DEFAULT_USER=your_rabbitmq_default_user
+RABBITMQ_DEFAULT_PASS=your_rabbitmq_default_password
+RABBITMQ_USER=your_rabbitmq_user
+RABBITMQ_PASSWORD=your_rabbitmq_password
+RABBITMQ_HOST=your_rabbitmq_host:port
+RABBITMQ_AUTH_QUEUE=your_auth_queue_name
+
+POSTGRES_USER=your_postgres_user
+POSTGRES_PASSWORD=your_postgres_password
+POSTGRES_DB=your_database_name
+
+POSTGRES_URI=postgresql://${POSTGRES_USER}:${POSTGRES_PASSWORD}@your_postgres_host:5432/${POSTGRES_DB}
```

Comment:
<div>


<div id="suggestion">
<div id="issue"><b>Consider direct env vars for POSTGRES_URI</b></div>
<div id="fix">

Consider using environment variables directly in the `POSTGRES_URI` instead of using string interpolation. This approach might be more secure and less prone to errors.
</div>
<details>
<summary>
<b>Code suggestion</b>
</summary>
<div id="code">

```
 @@ -12 +12 @@
 -POSTGRES_URI=postgresql://${POSTGRES_USER}:${POSTGRES_PASSWORD}@your_postgres_host:5432/${POSTGRES_DB}
 +POSTGRES_URI=postgresql://$POSTGRES_USER:$POSTGRES_PASSWORD@your_postgres_host:5432/$POSTGRES_DB
```
</div>
</details>
</div>


<small><i>Code Review Run <a href=https://github.com/omkardongre/messenger-api/pull/4#issuecomment-2439361771>#08fe39</a></i></small>
</div>

---
Is this a valid issue, or was it incorrectly flagged by the Agent?
- [ ] it was incorrectly flagged